### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by vbevziuk**
- [DEMO LINK](https://vbevziuk.github.io/layout_catalog/)
- [TEST REPORT LINK](https://vbevziuk.github.io/layout_catalog/report/html_report/)

tests show 4 errors and I don't understand why. Please indicate where I should look for the cause of the errors)

### CODE REVIEW

File: src/styles/blocks/product-cards.scss [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-end;
+    margin-bottom: 16px;
+
+    &-title {
+      color: $secondary-text-color;
+      font-size: 12px;
+      font-weight: 400;
```
Comment (sTorba24) [inline src/styles/blocks/product-cards.scss:63]: it's default value, can be removed
fix everywhere

File: src/styles/main.scss [hunk 2]
```diff
@@ -1,3 +1,38 @@
...
+  width: 100%;
+  border: solid $navitems-color 1px;
+  border-radius: 2px;
+  background-color: $navitems-color;
+}
+
+.main {
+  display: grid;
+  @include mainColumnsCount;
+  gap: 48px 46px;
```
Comment (sTorba24) [inline src/styles/main.scss:34]: swap these values, because first is vertical, second horizontal
mockup - vertical 46, horizontal - 48px

**Review by sTorba24 — CHANGES_REQUESTED** [review]
Almost good! Check button height, it stands out the most

**Review by VitaliyBondarenko1982 — APPROVED** [review]
Well done

**Pull request closed without merging**